api/envoy/config/route/v3/route.proto

```diff
@@ -135,6 +135,10 @@ message RouteConfiguration {
   // NOTE: this option will not strip the port number (if any) contained in route config
   // :ref:`envoy_v3_api_msg_config.route.v3.VirtualHost`.domains field.
   bool ignore_port_in_host_matching = 14;
+
+  // The policy that used to determine which name or value will be used as the searching key to get per filter config.
+  // [#not-implemented-hide:]
+  FilterConfigKeyPolicy filter_config_key_policy = 15;
```

Comment:
I didn't fully track the conversation that you were having with @markdroth, but I would really rather not add new configuration if we don't need to. Can we sketch out what a fallback policy would look like and see if that will cover known use cases? We can add config later if we really need to. Thank you!

/wait